### ISSUE TITLE
Search time

### ISSUE THREAD

**Issue by therese3** [yakir12/MAT2db.jl#40]
When I tracked the beetles, I stopped tracking them when they left the image, even though they came back a while later (but then I would have missed a part of their search). That means that the search time is quite variable. With the coffee beetles, we always let them search for 4 min. I think the short search time in some of the tests affects the centre of search point, am I right? What should we do? This feels like a discussion issue.

**Comment by yakir12** [comment]
I agree. I'm not sure what's best here. I'll make sure that the code accounts for these missing data points, but at the end of the day we can only keep in mind that *that* specific run is problematic because we are missing a part of its track. It is then a matter of choice to keep or exclude those runs from the overall analysis. We can have a general rule that if the part we miss is after the 3 minutes mark or if it's shorter than 30 seconds then we keep the run, otherwise, we discard it. I can program those kind of rules into the pipeline. 

I'll keep this open for now.

**Comment by therese3** [comment]
I will go through my notes and see what the optimal search time would be. I will get back to you about this.
